addons/google_account/models/google_service.py

```diff
@@ -120,6 +120,42 @@ def _get_google_token_json(self, authorize_code, service):
             error_msg = _("Something went wrong during your token generation. Maybe your Authorization Code is invalid")
             raise self.env['res.config.settings'].get_config_warning(error_msg)
 
+    @api.model
+    def _get_access_token(self, refresh_token, service, scope):
+        """Fetch the access token thanks to the refresh token."""
+        get_param = self.env['ir.config_parameter'].sudo().get_param
```

Comment:
This is a bit strange, we usually make a variable like "ConfigParameter" that holds the sudo'ed browserecord.
Not a big deal though.

Reply:
yes, but the the config parameter (especially in this file), we do like this (so I used the same as other methods in the same file)